src/pages/docs/security/users-and-teams/security-and-un-scoped-variables.md

```diff
@@ -9,7 +9,7 @@ navOrder: 1
 
 Sometimes, a team will be granted contributor access to a project, but be restricted in the environments that it can access. By default, Octopus's security system will then prevent members of the team editing [variables](/docs/projects/variables) that apply outside of their allowed environments. During development this can be inconvenient, as variables frequently need to be added in support of new application features.
 
-## Why restrict editing of unscoped variables? {#Securityandun-scopedvariables-Whyrestricteditingofun-scopedvariables?}
+## Why restrict editing of unscoped variables? {#why-restrict-editing-of-unscoped-variables}
```

Comment:
I followed the same approach Steve Fenton used in this previous PR where we encountered this: https://github.com/OctopusDeploy/docs/pull/2360/commits/2bd1155bb6a346f4ff1819565f39c3bc515f9a70#diff-1e8a102c0408e76c53dd207562893eff2122c73073bd5e5f1da9dbb7f399b467

And confirmed there's no anchor link references to these that needed updating.